### ISSUE TITLE
Resolution of snapshot

### ISSUE THREAD

**Issue by PrimozRome** [VinceG/vue-web-cam#28]
Is there a way to control output resolution of the snapshot image? 

My snapshots are now 640x480px, even though my webcam supports resolution up to 2560px...

**Comment by VinceG** [comment]
@PrimozRome I think it's based on the size of the canvas. Did you try making the canvas bigger?

**Comment by PrimozRome** [comment]
Well don't think so, because my canvas is smaller then 640x480 which is the resolution of image I get.... 

And I would not like to make canvas of the video object bigger as I have it now...

It would be cool if output resolution was somehow set up as props?

**Comment by VinceG** [comment]
@PrimozRome How do you even set the output resolution? We capture the image using:

this.getCanvas().toDataURL(this.screenshotFormat);

and looking at the API on mozilla i don't see any way to configure it.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL

Any idea how this is done?

**Comment by PrimozRome** [comment]
@VinceG I am using components actions to capture the image. I have a button and I call this on click event

```
takeSnapshot() {
  this.snapshot = this.$refs.webCam.capture()
},

```

this is how I feed the component

```
<web-cam
        v-show="!snapshot"
        ref="webCam"
        :deviceId="selectedDeviceId"
        :screenshot-format="format"
        @cameras="registerCameras"
        @started="streams.push($event)"
        height="auto"
/>

```

In your component you are setting the canvas size like this. Isn't it possible to have canvas width and height like parameters, instead of the size of video element? 

```
    getCanvas() {
      let video = this.$refs.video;
      if (!this.ctx) {
        let canvas = document.createElement("canvas");
        canvas.height = video.videoHeight;
        canvas.width = video.videoWidth;
        this.canvas = canvas;
        this.ctx = canvas.getContext("2d");
      }
      const { ctx, canvas } = this;
      ctx.drawImage(video, 0, 0, canvas.width, canvas.height);
      return canvas;
    }

```

**Comment by PrimozRome** [comment]
@VinceG what about this https://webrtc.github.io/samples/src/content/getusermedia/resolution/

**Comment by medvesekg** [comment]
The webcam resolution can be passed as a constraint with the [getUserMedia](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) call. I have submitted a pull request #29

**Comment by VinceG** [comment]
@PrimozRome @medvesekg Thank you. I've tagged and released it as v1.4.4.
